src/datascience-ui/interactive-common/variableExplorer.tsx

```diff
@@ -234,6 +278,13 @@ export class VariableExplorer extends React.Component<IVariableExplorerProps, IV
     }
 
     private renderGrid() {
+        let newGridHeight: number | undefined;
+
+        // In in view mode, just use the viewHeight prop for calculating size
+        if(this.props.viewMode) {
```

Comment:
Previously variableExplorer kept an internal state outside of redux for setting vertical height. For the new view mode we don't need this, so we can just look at the viewHeight prop for sizing.